plugin/manager.go

```diff
@@ -363,9 +363,14 @@ func (pm *Manager) initPlugin(p *plugin) error {
 	return err
 }
 
-func (pm *Manager) remove(p *plugin) error {
+func (pm *Manager) remove(p *plugin, force bool) error {
 	if p.PluginObj.Active {
-		return fmt.Errorf("plugin %s is active", p.Name())
+		if !force {
+			return fmt.Errorf("plugin %s is active", p.Name())
+		}
+		if err := pm.disable(p); err != nil {
+			logrus.Errorf("Error disabling plugin '%s': %s", p.Name(), err)
```

Comment:
Starting with "Error" is a bit redundant, perhaps

    unable to disable plugin ...